### ISSUE TITLE
V1: VPC configuration

### ISSUE THREAD

**Issue by nikgraf** [serverless/serverless#1489]
I couldn't find any VPC configuration in v1 (neither docs nor code)

**Comment by eahefnawy** [comment]
yeah it's currently not supported. I think you can do it by via CF though by adding resources to serverless.yaml

**Comment by eahefnawy** [comment]
This should be easy to implement:

```yml
functions:
    hello:
        handler: handler.hello
        vpc:
            securityGroupIds:
                - firstId
                - secondId
            subnetIds:
                - firstId
                - secondId
```
We'll just grab these configs and pass it to the Lambda CF template.

**Comment by flomotlik** [comment]
@pmuens @eahefnawy do you guys think we could get this in for the beta? It looks relatively simple and straight forward if we e.g. start by implementing this service wide and then potentially add the ability to set this per function as well. Thoughts?

**Comment by eahefnawy** [comment]
I can put it in right now :) ... Was keeping it for someone to `please-contribute` it 😄

**Comment by pmuens** [comment]
Great addition! Thanks for taking this @eahefnawy!

**Comment by svdgraaf** [comment]
I was just looking for this :D @eahefnawy if I can help with anything, let me know!

**Comment by flomotlik** [comment]
Awesome @svdgraaf will ping you tomorrow about this, we'd love to get this in by Beta next week. Btw would be great to chat with you sometime, I love talking to our contributors. Sent you an email to the one you provided in your github profile, if thats not the right one could you ping me at flo@serverless.com?

**Comment by svdgraaf** [comment]
@flomotlik done!

I have some time tomorrow to help out, some time in the weekend and at (least) a full day on Monday, so if I can help, let me know :) I actually need this feature for a project I'm working on (using AWS DirectConnect) so the sooner, the better ;)

**Comment by flomotlik** [comment]
Resolved with #1705
